java/create-save-map/README.md

```diff
@@ -4,23 +4,16 @@ Create a map and save it to your portal.
 ![Create and Save a Map App](create-save-map.png)
 
 ## How to use the sample
-This sample uses the [named user login](https://developers.arcgis.com/authentication/#named-user-login) authentication pattern.  Before a user can save a map they have to login to their [ArcGIS Online](https://www.arcgis.com/) account. Follow the [authentication](#authentication) steps below before running the app.  
+This sample uses the [named user login](https://developers.arcgis.com/authentication/#named-user-login) authentication pattern. Before a user can save a map they have to login to their [ArcGIS Online](https://www.arcgis.com/) account. Follow the steps below before running the app.  
 
-1. Open the navigation drawer by tapping on the Drawer Toggle or sliding right from the left side of the screen. Choose basemap and operational layers to create a map. Tap on save menu option to save the map. User will be provided with an authentication page to login.  Once logged in you provide a title, tags and description (optional) for the map.
-1. Login to your [ArcGIS Developers site](http://developers.arcgis.com) account and [Register](https://developers.arcgis.com/applications/#/new/) your app.  Once registered, select the **Authentication** tab taking note of your **Client ID**.
-1. A custom Redirect URI has been set up for this app, while still under the **Authentication** tab in your [ArcGIS Developers site](http://developers.arcgis.com) account, navigate down the page to the **Redirect URIs** section and add `my-ags-app://auth` redirect uri. 
-1. Open the **portal_settings.xml** resource file found in **..\src\res\values\** directory.  Edit the following with the `CLIENT_ID` value you obtained above:  
-	```xml
-	<!-- Client ID of your registered application --> 
-	<item name="client_id">your-clientID</item> 
-	```
+1. Open the navigation drawer by tapping on the Drawer Toggle or sliding right from the left side of the screen. 
+1. Choose basemap and operational layers to create a map. 
+1. Tap on save menu option on the right-hand side of the action bar to save the map.
+1. Enter a title, tags and description for your map and then hit 'Save Map'.
+1. Enter your credentials when prompted and your map will be saved to your [ArcGIS Online](https://www.arcgis.com/) account.
 
 ## How it works
-1. Create an `OAuthLoginManager`.
-1. Use it to `lauchOAuthBrowserPage`.
-1. Receive the resulting `Intent` in the MapSaveActivity.
-1. Fetch the OAuth token from the OAuthLogin Manager using `fetchOAuthTokenCredential(intent)`
-1. Set the OAuth credentials to a `Portal`.
+1. Create an `AuthenticationChallengeHandler` and set it to the `AuthenticationManager`.
```

Comment:
Since this is a one of the authentication samples, do you want to explain how to use the portal? I feel that the user would benefit from this information.